canonical-web-and-design/maas-ui
# Add user form

Done:
- Add user add and edit pages and form. Fixes: https://github.com/canonical-web-and-design/maas-squad/issues/1353.
- Added form, input, select, textarea, button and card components.

QA:
- open the users list
- click the edit pencil icon on a user
- you should see a user form like this: https://app.zeplin.io/project/5ce5423a25863c1e0d7abb27/screen/5d38490dfcd3ff77608afba4
